package.json

```diff
@@ -139,6 +139,7 @@
     }
   },
   "dependencies": {
+    "@cospired/i18n-iso-languages": "^4.1.0",
```

Comment:
This is used for mapping of language codes to their names to then feed for GOG's scriptinterpreter.exe. Normally we would've used `Intl` instead, however this is not present in nodejs < 20

Reply:
I tried with node 16 and the Intl module is present, I can do this with no issue https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames